docs/04.INTERNALS.md

```diff
@@ -247,9 +247,7 @@ Chunk size of the pool is 8 bytes (reduces fragmentation).
 ### Number
 
 There are two possible representation of numbers according to standard IEEE 754:
-
-* 4-byte (float, compact profile)
-* 8-byte (double, full profile)
+The default is 8-byte (double), but its able to use 4-byte (set CONFIG_ECMA_NUMBER_TYPE value), with the risk that it could lead inaccuracies.
```

Comment:
split this into two lines.
but the engine supports the 4-byte (single precision) representation by setting CONFIG_ECMA_NUMBER_TYPE as well